src/icesat2_tracks/config/__init__.py

```diff
@@ -0,0 +1 @@
+"Configuration file containing project paths"
```

Comment:
Docstrings are usually triple-quoted:
```suggestion
"""Configuration file containing project paths"""
```